webpack/scenes/ContentViews/Details/Filters/ContentViewFilters.js

```diff
@@ -48,7 +48,7 @@ const ContentViewFilters = ({ cvId }) => {
       if (filter.type === 'erratum' && filter.rules[0].types) errataByDate = true;
 
       const cells = [
-        { title: <Link to={cvFilterUrl(cvId, id)}>{name}</Link> },
+        { title: (type === 'package_group' || type === 'rpm') ? <Link to={cvFilterUrl(cvId, id)}>{name}</Link> : name },
```

Comment:
@jeremylenz : Updated so the package_group and rpm filters are links to detail page and the rest are not links, just names.